app/liquid_tags/codesandbox_tag.rb

```diff
@@ -1,15 +1,17 @@
 class CodesandboxTag < LiquidTagBase
   PARTIAL = "liquids/codesandbox".freeze
+  REGISTRY_REGEXP = %r{https?://(?:www|app\.)?(?:codesandbox\.io/embed/)(?<video_id>[\w-]{,60})(?:\?)?(?<options>\S+)?}
   OPTIONS_REGEXP =
     %r{\A(initialpath=([a-zA-Z0-9\-_/.@%])+)\Z|
+      \A(file=([a-zA-Z0-9\-_/.@%])+)\Z|
       \A(module=([a-zA-Z0-9\-_/.@%])+)\Z|
       \A(runonclick=((0|1){1}))\Z|
       \Aview=(editor|split|preview)\Z}x
 
   def initialize(_tag_name, id, _parse_context)
     super
-    @id = parse_id(id)
-    @query = parse_options(id)
+    input       = CGI.unescape_html(strip_tags(id))
```

Comment:
Is the `CGI.unescape_html` needed? To be clear, I don't think it will cause any harm, Despite my best efforts (srsly, I tried), I couldn't get it to inject arbitrary attributes to [the `iframe`](https://github.com/forem/forem/blob/4b93684dab5714a1dc1aab7987510743d0e53d2b/app/views/liquids/_codesandbox.html.erb#L2) no matter what I threw at it since the id matcher specifies that the `id` must be 0-60 alphanumeric/hyphen characters, so big ups on that!

Reply:
During my local testing, some URLs came in with `&amp;` characters when the input had options as parameters. So in my opinion, I would rather be safe than sorry.